packages/framer-motion/src/render/VisualElement.ts

```diff
@@ -732,7 +732,7 @@ export abstract class VisualElement<
             this.addValue(key, value)
         }
 
-        return value as MotionValue
+        return value!
```

Comment:
Any reason why we can't typecheck it here? We could throw an explicit error or add an assert, right?

Reply:
Will revert from this PR and have fixed in https://github.com/framer/motion/pull/1843